xbmc/addons/kodi-dev-kit/include/kodi/c-api/gui/general.h

```diff
@@ -26,6 +26,7 @@ extern "C"
     int (*get_current_window_dialog_id)(KODI_HANDLE kodiBase);
     int (*get_current_window_id)(KODI_HANDLE kodiBase);
     ADDON_HARDWARE_CONTEXT (*get_hw_context)(KODI_HANDLE kodiBase);
+    int (*get_adjust_refresh_rate)(KODI_HANDLE kodiBase);
```

Comment:
If the `enum` is in this header, can also the function return as `enum AdjustRefreshRateStatus (*get_adjust_refresh_rate)(KODI_HANDLE kodiBase);` (maybe also with `_status` on end of function name to have clear.

With the `int` and as name "get adjust refresh rate" sounds confusing as outsiders might expect an integer value of the active refresh rate.

Reply:
done